R/g.report.part4.R

```diff
@@ -76,6 +76,9 @@ g.report.part4 = function(datadir = c(), metadatadir = c(), loglocation = c(),
     }
     nightsummary2 = as.data.frame(do.call(rbind, lapply(fnames.ms4, myfun)), stringsAsFactors = FALSE)
     nightsummary2$night = as.numeric(gsub(" ", "", nightsummary2$night))
+    nightsummary2$calendar_date = as.Date(nightsummary2$calendar_date, format = "%d/%m/%Y")
```

Comment:
This makes a hard assumption about date format. Did you test this with computer configured in different date formats?

Reply:
This calendar date is taken from `g.create.sp.mat`, which uses this in [line 35](https://github.com/wadpac/GGIR/blob/master/R/g.create.sp.mat.R):

`calendar_date_safe = paste(soso$mday,"/",(soso$mon+1),"/",(soso$year+1900),sep="")`

That's why in part 4 reports, the calendar date always look as `"%d/%m/%Y"`, independently of the system default format.